framework/wazuh/core/cluster/server.py

```diff
@@ -17,15 +17,28 @@
 import uvloop
 
 from wazuh.core import common, exception, utils
-from wazuh.core.cluster import common as c_common
+from wazuh.core.cluster.common import asyncio_exception_handler, Handler
 from wazuh.core.cluster.utils import ClusterFilter, context_tag
+from wazuh.core.cluster.common import Handler
 
 
-class AbstractServerHandler(c_common.Handler):
+class AbstractServerHandler(Handler):
     """
     Define abstract server protocol. Handle communication with a single client.
     """
 
+    def _create_cmd_handlers(self):
+        """
+            Add command handlers to _cmd_handler dictionary
+        """
```

Comment:
Review this everywhere https://numpydoc.readthedocs.io/en/latest/format.html#short-summary

Also, move it after `__init__` method in all the modules.

Reply:
Moved after __init__ and fixed Docstring